a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -292,6 +296,12 @@ def get_update_load_balancer_flow(self):
             requires=constants.LOADBALANCER))
         update_LB_flow.add(vthunder_tasks.WriteMemory(
             requires=a10constants.VTHUNDER))
+        update_LB_flow.add(a10_database_tasks.MarkVThunderStatusInDB(
```

Comment:
Added this task, because whenever update lb is executed, vthunder goes to PENDING_UPDATE state, and is not set to ACTIVE after success.
Checked the same with other slb flows, but seems to be problem with lb flow only. Hence added this task.